tests/api/exclude/pyiceberg-0.6.0.yaml

```diff
@@ -0,0 +1,47 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# The format of this file is documented at
+# https://cwiki.apache.org/confluence/display/INFRA/Git+-+.asf.yaml+features
+
+exclude:
+- obj_path: pyiceberg.avro.decoder_fast.CythonBinaryDecoder
+  kind: CLASS_REMOVED_BASE
+- obj_path: pyiceberg.table.create_mapping_from_schema
+  kind: OBJECT_REMOVED
+- obj_path: pyiceberg.avro.decoder_fast.BinaryDecoder
+  kind: OBJECT_REMOVED
+- obj_path: pyiceberg.avro.decoder_fast.CythonBinaryDecoder.tell
```

Comment:
Alright, so it looks like this false positive is being generated because we are using two separate approaches for loading the Package in griffe. We use `load_git` to load pyiceberg's latest release directly from Git, and then we use `load` to load the local implementation of pyiceberg with the coder's current change. Tracking the outputs of CythonBinaryDecoder for the two methods yields different interpretations of the Cython class's interface:

```
>>> current_git = griffe.load_git("pyiceberg")
>>> current_git.all_members['avro']['decoder_fast']['CythonBinaryDecoder'].all_members
{'__init__': Function('__init__', 21, 22), 'tell': Function('tell', 24, 25), 'read': Function('read', 27, 28), 'read_boolean': Function('read_boolean', 30, 31), 'read_int': Function('read_int', 33, 34), 'read_ints': Function('read_ints', 36, 37), 'read_int_bytes_dict': Function('read_int_bytes_dict', 39, 40), 'read_bytes': Function('read_bytes', 42, 43), 'read_float': Function('read_float', 45, 46), 'read_double': Function('read_double', 48, 49), 'read_utf8': Function('read_utf8', 51, 52), 'skip': Function('skip', 54, 55), 'skip_int': Function('skip_int', 57, 58), 'skip_boolean': Function('skip_boolean', 60, 61), 'skip_float': Function('skip_float', 63, 64), 'skip_double': Function('skip_double', 66, 67), 'skip_bytes': Function('skip_bytes', 69, 70), 'skip_utf8': Function('skip_utf8', 72, 73)}
>>> current_local = griffe.load("pyiceberg")
>>> current_local.all_members['avro']['decoder_fast']['CythonBinaryDecoder'].all_members
{'__doc__': Attribute('__doc__', None, None), '__new__': Function('__new__', None, None), '__pyx_vtable__': Attribute('__pyx_vtable__', None, None), '__reduce__': Function('__reduce__', None, None), '__setstate__': Function('__setstate__', None, None), 'read': Function('read', None, None), 'read_boolean': Function('read_boolean', None, None), 'read_bytes': Function('read_bytes', None, None), 'read_double': Function('read_double', None, None), 'read_float': Function('read_float', None, None), 'read_int': Function('read_int', None, None), 'read_int_bytes_dict': Function('read_int_bytes_dict', None, None), 'read_ints': Function('read_ints', None, None), 'read_utf8': Function('read_utf8', None, None), 'skip': Function('skip', None, None), 'skip_boolean': Function('skip_boolean', None, None), 'skip_bytes': Function('skip_bytes', None, None), 'skip_double': Function('skip_double', None, None), 'skip_float': Function('skip_float', None, None), 'skip_int': Function('skip_int', None, None), 'skip_utf8': Function('skip_utf8', None, None), 'tell': Function('tell', None, None)}
```

So it looks like only the local `load` is reading the pyx file, whereas load_git is reading the pyi file.

Maybe we exclude cython APIs from the test while we figure out how to resolve this issue?